tw-NEDa-frontend/src/app/app.component.spec.ts

```diff
@@ -1,13 +1,12 @@
 /* tslint:disable:no-unused-variable */
```

Comment:
Fix the formatting & code style. Once the formatter is the same this file should not appear as something that changed.